### PR TITLE
v1.4.3 - Update Login Method and fix Login Problem with several Servers like EU

### PR DESCRIPTION
**Pull request by lukkie112**
- Update Login Method and fix Login Problem with several Servers like EU
  1. After Installation, open pCloud Settings
  2. At first Tab choose for US Server https://api.plcoud.com and for EU Server https://eapi.plcoud.com
  3. Than Enter in the correct Tab (api or eapi) your Username and Password and click OK
  4. Start pCloud and have fun
  Thx to loki1979

### CODE REVIEW

**Comment by gdomenici** [comment]
Many thanks for the PR @lukkie112 . As soon as I have some contiguous time I'll review it properly and, if all OK, will merge. On first glance, everything looks OK though.
Kind regards,
Guido

File: resources/settings.xml [hunk 1]
```diff
@@ -2,10 +2,16 @@
...
     <category label="30100">
 		<!-- Will contain the pcloud auth string, but is only set programmatically -->
-        <setting label="30105" type="text" id="auth" visible="false" default=""/>
-		<setting label="30106" type="text" id="authExpiry" visible="false" default=""/>
-		<!-- Last used folder ID -->
-		<setting label="30112" type="text" id="lastUsedFolderID" visible="false" default="0"/>
+		<setting label="CLOUD URL" type="labelenum" id="CloudUrl" values="https://api.pcloud.com/|https://eapi.pcloud.com/" default="https://api.pcloud.com/"/>
+		<setting type="sep"/>
+		<setting label="API USERNAME" type="text" id="username1" default=""/>
+        <setting label="API PASSWORD" type="text" id="password1" default=""/>
```
Comment (gdomenici) [inline resources/settings.xml:8]: Same comment here - I wouldnt store the username and pwd in the config file

**Review by gdomenici — CHANGES_REQUESTED** [review]
Please consider changing the code so that we dont store the actual creds in the config file

**Comment by lukkie112** [comment]
I will ask my friend,if he can make the changes you would like

**Comment by gdomenici** [comment]
> I will ask my friend,if he can make the changes you would like

Hi @lukkie112 

Thanks for that.

Another possibility (an alternative that may make your life easier) is the following:

1. You branch the code in your repo from `master` to another branch name. For example (just an example) `eapi-access-changes`.
1. you cancel this current PR.
1. then you do a _new_ PR with the existing code (no changes) from that new branch to a same-named branch in my repo.
1. then I (not you) merge that PR to that new branch in my repo.
1. then I do the necessary modifications to your code myself
1. and when I'm done I merge to my `master`.

That would make your life easier. Or, of course, you can proceed as you propose above, i.e. your friend does the necessary changes. (But even in that case, i'd recommend using a new branch name). Up to you.

Kind regards,

Guido

**Comment by lukkie112** [comment]
I was doing an new PullRequset with a new branche: Pcloud eu us login #37

So I close this one here

**Pull request closed without merging**